### PR TITLE
Chore(img): Reduce DropZone image file size

### PR DESCRIPTION
**Pull request by samuellye98**
Replace dropzone image with html text

### CODE REVIEW

File: src/components/HomePageContent/DropZoneSection.js [hunk 1]
```diff
@@ -44,7 +44,15 @@ const DraggableDemoCertificate = () => (
       </div>
       <div className="col">
         <img
-          src="/static/images/dropzone/arrow3.png"
+          src="/static/images/dropzone/arrow.png"
+          width="100%"
+          draggable="false"
+        />
+        <h8 style={{ fontSize: 14, textAlign: "justify", color: "#ff9933" }}>
```
Comment (Nebulis) [inline src/components/HomePageContent/DropZoneSection.js]: h8 dont exist in native browser tag :D

Rendering changed quite a lot
from 
image

to
image

**Comment by Nebulis** [comment]
Icon size can probably be lowered more, however I dunno if it's really a concern:


This is the size of the loading site
image

File: src/components/HomePageContent/DropZoneSection.js [hunk 2]
```diff
@@ -44,7 +44,22 @@ const DraggableDemoCertificate = () => (
...
       <div className="col">
         <img
-          src="/static/images/dropzone/arrow3.png"
+          src="/static/images/dropzone/arrow.png"
+          width="100%"
+          draggable="false"
+        />
+        <span
+          style={{
+            // fontSize: "15px",
```
Comment (Nebulis) [inline src/components/HomePageContent/DropZoneSection.js]: to remove :p

**Comment by thefiend** [comment]
Probably want to replace the "/static/images/dropzone/opencerts.png" with "/static/images/opencertslogo.svg" so that there is one less image and the svg image is cleaner.

Screenshot 2019-07-25 at 12 57 09 AM

**Pull request merged**